src/common/components/VoucherNumber/VoucherNumberValue.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue } from '@folio/stripes/components';
+
+const VoucherNumberValue = ({ value }) => (
+  <KeyValue
+    label={<FormattedMessage id="ui-invoice.invoice.details.voucher.voucherNumber" />}
+    value={value}
```

Comment:
The whole sense to leverage code to such components was to use same behavior for displaying values. Like displaying <NoValue /> if value is not provided. If you make this just like a proxy - it's over-engineering. For this very case I don't see any reason to not use <NoValue /> in case value is not provided